shared/components/NextCommunityEvent/index.js

```diff
@@ -56,25 +80,7 @@ const NextCommunityEvent = ({
             </li>
           </ul>
         </div>
-        <div className="NextCommunityEvent__section NextCommunityEvent__section__booking">
-          <h3 className="NextCommunityEvent__booking__heading">This event has ended</h3>
-          <p className="NextCommunityEvent__live-stream-text">
-            Couldn’t make the event? <div>We’ve got your back.</div>
-          </p>
-          <div className="NextCommunityEvent__booking-btn__container">
-            <a
-              className="NextCommunityEvent__booking-btn NextCommunityEvent__booking-btn--active"
-              href="https://www.youtube.com/watch?v=HrECWxWVcEI"
-            >
-              Watch Video
-            </a>
-          </div>
-          <p className="NextCommunityEvent__live-stream-text">
-            To get reminders about tickets and future
-            events <a className="NextCommunityEvent__live-stream-text--link" href="#stay-tuned">
-            subscribe here</a>
-          </p>
-        </div>
+        <TicketStatus />
```

Comment:
Discussed offline. Decided to call it EventStatus but this is a lower priority change - perhaps even in a new PR as this guy is a monolith